src/kili/queries/project_user/__init__.py

```diff
@@ -43,7 +44,7 @@ def project_users(
         skip: int = 0,
         disable_tqdm: bool = False,
         as_generator: bool = False,
-    ) -> Union[List[dict], Generator[dict, None, None]]:
+    ):
```

Comment:
why removing the return type ?

Reply:
it is an error, I put in back with `Iterables[Dict]` as it is for other queries